Coop-board-resolution-transparency-report.md

```diff
@@ -1,3 +1,8 @@
+## Board Resolution Transparency Report
+
+The table below reflects all published RChain board resolutions since 201709 with their respective votes from each board member. Exported from source [spreadsheet](https://docs.google.com/spreadsheets/d/1XnSyGNXoQmpvS8hgTFWZMZIC26tQQmdDjD8RKP0msa8/edit#gid=1001645341) 20181026.
```

Comment:
Great, thanks! I would be useful to know if there are _unpublished_ board resolutions as well, of if this document describes all resolutions on which votes were taken.

Reply:
I know that @n10n can answer that question.

Reply:
Who owns the google doc?